src/app/shared/model/invalid/template.invalid.ts

```diff
@@ -0,0 +1,66 @@
+import { AudioVersionTemplateModel } from '../audio-version-template.model';
+import { AudioFileTemplateModel } from '../audio-file-template.model';
+import { BaseInvalid } from './base.invalid';
+
+/**
+ * Audio version template length
+ */
+const checkMinimum = (min, max): string => {
+  if (min === null && max === null) {
+    return null; // allow unset
+  } else if (min === null) {
+    return `Must set a minimum`;
+  } else if (isNaN(parseInt(min, 10))) {
+    return `Minimum is not a number`;
+  } else if (min < 1) {
+    return `Minimum must be greater than 0`;
+  } else if (min >= max) {
+    return 'Minimum must be less than maximum';
```

Comment:
That's currently how the server validation works - if either of them are set, both must be set.  I'm also guessing that most users will fill in both numbers, since they'll have a pretty good idea of sane lengths for their show.

If that's not the case, could do something like send a really-big-number if max is missing (and send 0 if min is missing).  Or... change the server code.